### PR TITLE
Ahruygt/4555 installer background

### PR DESCRIPTION
**Pull request by ahruygt**
Added 1x and 2x images to installer background.

### CODE REVIEW

**Comment by larkost** [comment]
@VeXocide: would you mind looking at the code part of this?

File: packaging/osx/Distribution.xml [hunk 1]
```diff
@@ -8,7 +8,7 @@
 
 <options customize="never" allow-external-scripts="no" rootVolumeOnly="yes" />
 
-<background file="installer_background.png" mime-type="image/png" alignment="topleft" scaling="none" />
+<background file="installer_background.tiff" mime-type="image/png" alignment="topleft" scaling="none" />
```
Comment (VeXocide) [inline packaging/osx/Distribution.xml]: The `mime-type` is no longer correct, it should be `image/tiff`.

File: packaging/osx/create_dmg.py [hunk 2]
```diff
@@ -111,11 +111,34 @@ def buildPackage(versionString, serverRootPath, installPath=None, signingName=No
...
+		sys.stderr.write('Failed while building installer background: %s\n%s' % (str(e), logFile.read()))
+		raise
+	
 	# == assemble the archive
 	
 	distributionPath = os.path.join(scratchFolder, 'rethinkdb-%s.pkg' % versionString)
 	
-	productBuildCommand = ['/usr/bin/productbuild', '--distribution', os.path.join(thisFolder, 'Distribution.xml'), '--package-path', packageFolder, '--resources', os.path.join(thisFolder, 'installer_resources'), distributionPath]
+	productBuildCommand = [
+	   '/usr/bin/productbuild',
```
Comment (VeXocide) [inline packaging/osx/create_dmg.py]: This mixes tabs and spaces.

**Comment by VeXocide** [comment]
@larkost -- looks good now

**Comment by ahruygt** [comment]
Retina : 
retina-installer-bg

Non Retina:
non-retina-installer-bg

**Comment by mglukhovsky** [comment]
Looks good to me!

**Pull request merged**